frontend/src/store/modules/auth/actions.js

```diff
@@ -111,4 +111,30 @@ export default {
             });
         });
     },
+    saveUser: (context, formData) => {
```

Comment:
updateProfile(context, userId, formData) is more clear

Reply:
I can not use (context, userId, formData) because the method can only accept one attribute